Gemfile

```diff
@@ -1,7 +1,6 @@
 source "https://rubygems.org"
 
 # Bundle edge Rails instead: gem "rails", github: "rails/rails"
-gem "arel", "~> 9"
```

Comment:
Minor: would be nice to provide a link in the commit message, showing where it says Arel is included in ActiveRecord.

Reply:
Good catch, I'll dig out something from the docs.